full_install.sh

```diff
@@ -1,12 +1,6 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-# Get the directory of the script
-script_dir=$(dirname "$0")
-
-# Change the current directory to the script's directory
-cd "$script_dir"
-
```

Comment:
Maybe instead we only do this logic if not in a colab environment, instead of removing it?

Alternative, don't remove it, but if colab, force `--non-editable` flag setting. This should also fix the issue, since there is no need to have editable install in colab.